spikeinterface/postprocessing/spike_amplitudes.py

```diff
@@ -78,11 +78,10 @@ def _run(self, **job_kwargs):
         amps = np.concatenate(amps)
         segments = np.concatenate(segments)
 
-        self.amplitudes = []
+        amplitudes = []
```

Comment:
I think we should keep the amplitudes at a property no ?

Reply:
They are already in the extension_data